src/algebra/quandle.lean

```diff
@@ -0,0 +1,586 @@
+/-
+Copyright (c) 2020 Kyle Miller. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller
+-/
+import algebra
```

Comment:
I'm a little skeptical this file `src/algebra/default.lean` even needs to exist in the first place. Perhaps replace by what you actually need?

Reply:
It turns out `data.equiv.mul_add` imports everything that was needed about groups, so I removed this line.